functions/isLeap.js

```diff
@@ -31,4 +35,18 @@ output: true
 
 // YOUR CODE HERE
 
+function isLeap(year){
+    if(!isDivisibleBy(year,4)){
+        return false;
+    } else if (!isDivisibleBy(year,100)){
+        return true;
+        } else if (isDivisibleBy(year,4) && isDivisibleBy(year,100) && isDivisibleBy(year,400)){
+            return true;
+        } else {
+            return false;
+        }
+}
+
+
+
```

Comment:
I think you might one of the two (so far) that have actually utilized the isDivisibleBy function in the isLeap function - well done 👍